plugins/catalog-import/src/components/ImportInfoCard/ImportInfoCard.tsx

```diff
@@ -19,15 +19,26 @@ import { configApiRef, useApi } from '@backstage/core-plugin-api';
 import { Chip, Typography } from '@material-ui/core';
 import React from 'react';
 import { catalogImportApiRef } from '../../api';
+import { useCatalogFilename } from '../../hooks';
 
-export const ImportInfoCard = () => {
+type Props = {
+  exampleEntityFilepath?: string;
+  exampleRepositoryPath?: string;
```

Comment:
Alright, discussed, and we discovered one thing that we think is worth tweaking. What do you think about calling these `exampleLocationUrl` and `exampleRepositoryUrl` instead?

Reply:
I'm fine with the suggested name changes 👍

Reply:
and done